src/main/java/beam/physsim/cch/GraphReader.scala

```diff
@@ -0,0 +1,84 @@
+package beam.physsim.cch
+import java.io.File
+import java.nio.{ByteBuffer, ByteOrder}
+
+import com.google.common.io.Files
+import com.vividsolutions.jts.geom.Coordinate
+
+import scala.collection.mutable
+
+case class Vertex(id: Long, coordinate: Coordinate)
+case class RoutingFrameworkGraph(vertices: Seq[Vertex])
+
+trait RoutingFrameworkGraphReader {
+  def read(graph: File): RoutingFrameworkGraph
+}
+
+class RoutingFrameworkGraphReaderImpl extends RoutingFrameworkGraphReader {
+  override def read(graph: File): RoutingFrameworkGraph = {
+
+    val source = Files.asByteSource(graph).openStream()
```

Comment:
nitpick: source will left open in case of exception, would suggest to do it and try/finally

Reply:
fixed